### PR TITLE
pybind/mgr: add more modules to flake8

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/alerts/__init__.py [hunk 1]
```diff
@@ -1 +1,5 @@
 from .module import Alerts
+
+__all__ = [
+    'Alerts',
+]
```
Comment (sebastian-philipp) [inline src/pybind/mgr/alerts/__init__.py:5]: we need `__all__` if we are a library and our users are importing `from alerts import *`. I would add `# flake8: noqa` at the top of the file

File: src/pybind/mgr/balancer/__init__.py [hunk 2]
```diff
@@ -1 +1,6 @@
 from .module import Module
+
+
+__all__ = [
+    'Module'
+]
```
Comment (sebastian-philipp) [inline src/pybind/mgr/balancer/__init__.py:6]: I'd prefer `# flake8: noqa`

File: src/pybind/mgr/crash/__init__.py [hunk 3]
```diff
@@ -1 +1,5 @@
 from .module import Module
+
+__all__ = [
+    'Module'
+]
```
Comment (sebastian-philipp) [inline src/pybind/mgr/crash/__init__.py:5]: I'd prefer `# flake8: noqa`

File: src/pybind/mgr/devicehealth/__init__.py [hunk 4]
```diff
@@ -1,2 +1,5 @@
-
 from .module import Module
+
+__all__ = [
+    'Module'
+]
```
Comment (sebastian-philipp) [inline src/pybind/mgr/devicehealth/__init__.py:5]: I'd prefer `# flake8: noqa`

**Review by sebastian-philipp — APPROVED** [review]

**Comment by tchaikov** [comment]
@sebastian-philipp flake8-noqa'ed and repushed.

**Review by epuertat — APPROVED** [review]
👍🏼

File: src/pybind/mgr/crash/module.py [hunk 5]
```diff
@@ -42,13 +42,13 @@ class Module(MgrModule):
         Option(
             name='warn_recent_interval',
             type='secs',
-            default=60*60*24*14,
+            default=60 * 60 * 24 * 14,
```
[the remark this answers is not in the document]
Comment (tchaikov) [reply src/pybind/mgr/crash/module.py:45]: the return value of `total_seconds()`. and the underlying type of "secs" is `int`. so i'd keep as it is.

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2021-02-14_08:35:02-rados-wip-kefu-testing-2021-02-14-1248-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2021-02-16_03:53:21-rados-wip-kefu-testing-2021-02-15-2239-distro-basic-smithi/

cephadm failures are not related.

**Pull request merged**